message_definitions/v1.0/common.xml

```diff
@@ -6195,6 +6195,11 @@
       <field type="uint32_t" name="active" display="bitmask">Active outputs</field>
       <field type="float[32]" name="actuator">Servo / motor output array values. Zero values indicate unused channels.</field>
     </message>
+    <message id="376" name="RELAY_STATUS">
+      <description>Reports the on/off state of relays, as controlled by MAV_VMD_DO_SET_RELAY.</description>
+      <field type="uint64_t" name="time_usec" units="us">Timestamp (since system boot).</field>
+      <field type="uint32_t" name="states" display="bitmask">Relay states.  Relay instance numbers are represented as individual bits in this mask by offset.</field>
```

Comment:
maybe 2 uint16_t, one for which relays exist, one for which are currently high

Reply:
Done